### DISCUSSION TITLE
Look-Up Tables in XML File

### DISCUSSION THREAD

**Discussion by Flight2IC** [JSBSim-Team/jsbsim#1103] [General]
Hi everyone,
I want to make some changes in **f16.xml** file but having a hard time figuring out the name of **output property** in individual **table**. Input parameters and their units are provided but for output only the value is given. I have been through the JSBSim reference manual but that as well does not explains each table. 
Any help will appreciated.

**Comment by seanmcleod** [comment]
Here is an example, output property is the name of the function.

https://github.com/JSBSim-Team/jsbsim/blob/e33e184f512449b8120769825b20e4d35df8ee51/aircraft/737/737.xml#L453-L470

**Reply by Flight2IC** [reply]
Thank you so much for this valuable response. It helped and saved a lot of time.
